lib/pymedphys/_experimental/plancomplexity/operations.py

```diff
@@ -1,3 +1,5 @@
+# ruff: noqa: E741

```

Comment:
Why not just correct the variable name? `leaf` instead of `l`?

Reply:
Essentially I did a "if in _experimental I am not going to do maintenance work on it"